.github/workflows/publish-docker.yml

```diff
@@ -26,8 +24,6 @@ jobs:
         image:
           - name: cephlet
             target: manager
-          - name: populator
-            target: populator
           - name: ori-volume
```

Comment:
Is this the correct image name?

Reply:
the populator has been removed and is now covered by the `Snapshot` controller